plugins/Bookmark.jsx

```diff
@@ -18,7 +18,9 @@ import Spinner from '../components/Spinner';
 import {createBookmark, getUserBookmarks, removeBookmark, updateBookmark} from '../utils/PermaLinkUtils';
 import './style/Bookmark.css';
 import isEmpty from 'lodash.isempty';
-
+import { zoomToPoint } from '../actions/map';
+import { resolveBookmark } from '../utils/PermaLinkUtils';
+import MapUtils from '../utils/MapUtils';
 
```

Comment:
Please keep imports sorted (first absolute, then relative, and lastly the style import)

Reply:
Do we have some eslint rule for that ?

Reply:
Currently not, I suppose we could enable https://eslint.org/docs/latest/rules/sort-imports

Reply:
Ok I was looking to the same link. I can take the task ;-)